packages/mobile/scripts/run_e2e.sh

```diff
@@ -112,7 +116,7 @@ runTest() {
     --retries $RETRIES \
     --headless \
     "${test_match}" \
-    --reuse
+    "${extra_param}" 
```

Comment:
just checking you still want to make this change, if we are opting to disable Sentry for e2e?

Reply:
We still want this change. We never want CI to use a previous build; this was why the build was passing but then failing later after merging into main.